app/helpers/record_helper.rb

```diff
@@ -98,17 +98,23 @@ def access_type(metadata)
     access_right.first['description']
   end
 
+  # This method is not currently in use, but it may become useful later. Stakeholders need to see all available data
+  # for testing.
   def issued_dates(dates)
     return_relevant_dates(dates, 'Issued')
   end
 
+  # This method is not currently in use, but it may become useful later. Stakeholders need to see all available data
+  # for testing.
   def coverage_dates(dates)
     return_relevant_dates(dates, 'Coverage')
   end
 
+  # This method is likely to change post-MVP. Stakeholders need to see all available data for testing.
   def more_info?(metadata)
-    if issued_dates(metadata['dates']) || coverage_dates(metadata['dates']) || places(metadata['locations']) ||
-       metadata['provider']
+    if metadata['citation'] || metadata['dates'] || metadata['identifiers'] || metadata['languages'] ||
```

Comment:
When we come back to this, let's see if we can find a way to refactor this so it isn't a super long conditional.

I'm okay with going with this for now, but as your comment notes we'll come back to it let's rethink how to approach it at that time.